ocf_datapipes/transform/xarray/normalize.py

```diff
@@ -22,6 +22,8 @@ def __init__(
         max_value: Optional[Union[int, float]] = None,
         calculate_mean_std_from_example: bool = False,
         normalize_fn: Optional[Callable] = None,
+        min_values: Optional[np.ndarray] = None,
+        max_values: Optional[np.ndarray] = None,
```

Comment:
I think the naming is a bit confusing given the `max_value` is already there and is quite different; also I'm pretty sure they are `xr.DataArray`-s?

Reply:
Good point

Reply:
Ive updated it now